docs/source/installing_docker.rst

```diff
@@ -68,6 +68,20 @@ configuration of a new Kiwi TCMS installation::
 
 .. versionadded:: 8.9
 
+If you prefer web-based setup you can go to
+``https://<your_domain_or_ip>/init-db/``
+You will be redirected to this URL if your database is not initialised.
+After performing migrations you can register the first user by using the
+registration page:
+``https://<your_domain_or_ip>/accounts/register/``
+
+.. note::
+
+    The first registered user is automatically granted superuser rights and
+    is activated without mail verification.
+
+The next step is :ref:`configure-kiwi-domain`.
+
```

Comment:
The sequence of actions and documentation doesn't make logical sense. Assuming you start with a fresh system you need the minimum amount of documentation in order to get the system up and running. Which already is:

1) web approach: Go to https://localhost (or FQDN) and follow the prompts or 

cli approach:
2) Use the initial_setup cli command which is comprised of migrate + createsuperuser + set_domain + refresh_permissions. <- the command which we want people to use is initial_setup. The rest should be mentioned without too much information. It is expected that if you are going to use them you know what they are. If you don't stick to initial_setup.

Notes:
refresh_permissions is documented in admin.rst
set_domain is documented in the next section

createsuperuser isn't documented from what I can see.

In addition all commands have a 1 line documentation in admin.rst as well so we don't have to repeat that too much.